apps/site/lib/site/application.ex

```diff
@@ -8,6 +8,10 @@ defmodule Site.Application do
   def start(_type, _args) do
     import Supervisor.Spec, warn: false
 
+    if System.get_env("ALLOW_INDEXING") == "true" do
+      Application.put_env(:site, :allow_indexing, true)
+    end
```

Comment:
For codecov purposes, you could use into a helper function which takes the value of `System.get_env("ALLOW_INDEXING")` and returns the value to put in the environment?